MdeModulePkg/Application/BootManagerMenuApp/BootManagerMenu.c

```diff
@@ -454,10 +455,12 @@ BootMenuSelectItem (
   // Print want to select item
   //
   FirstItem = BootMenuData->ScrollBarControl.FirstItem;
-  gST->ConOut->SetAttribute (gST->ConOut, EFI_WHITE | EFI_BACKGROUND_BLACK);
+  gST->ConOut->SetAttribute (gST->ConOut, EFI_YELLOW | EFI_BRIGHT | EFI_BACKGROUND_BLUE);
   String = HiiGetString (gStringPackHandle, BootMenuData->PtrTokens[WantSelectItem], NULL);
   PrintCol = StartCol  + 1;
   PrintRow = StartRow + TITLE_TOKEN_COUNT + 2 + WantSelectItem - FirstItem;
+  PrintStringAt (PrintCol, PrintRow, L"→");
+  PrintCol = PrintCol + 2;
   PrintStringAt (PrintCol, PrintRow, String);

```

Comment:
```suggestion
  PrintStringAt (PrintCol + 2, PrintRow, String);
```

Maybe do this for consistency

Reply:
Comitted